microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer.md

```diff
@@ -0,0 +1,58 @@
+---
+title: View formatted JSON
+description: How to use the JSON viewer to view formatted and highlighted JSON responses in a browser tab.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/04/2023
+---
+# View formatted JSON
+
+Use the **JSON viewer** to automatically format and highlight JSON responses and files in browser tabs.
+
+When your web server responds to HTTP requests with data encoded as JSON, this data isn't always easy to read and is sometimes returned as one line of text. The **JSON viewer** changes the returned data to make it easier to read. The JSON syntax is highlighted with different colors, object properties are displayed on their own lines and indented, and objects can be collapsed or expanded.
+
+The **JSON viewer** tool is included in Microsoft Edge starting with version 110.
+
+
+<!-- ====================================================================== -->
+## View formatted JSON server responses
+
+To view a formatted JSON response from a web server:
```

Comment:
I see one risk with this: the tool does not reformat the actual JSON on the web server. It only shows a reformatted version of it, leaving the original data untouched. I know this may sound obvious to many, but it becomes less obvious when you open local JSON files from your disk.

"To reformat the JSON data that's in a file on disk" sounds very much like the tool actually reformats the file itself.

How about something in between:

* "To view a JSON response from a web server as reformatted JSON", and
* "To view a local JSON file as reformatted JSON".

Reply:
"local JSON file" is a little surprising to me, is it really a file on the local drive at this point?

Changing my suggestion from:
To reformat the JSON data that's returned by a web server
to:
To view a JSON response from a web server as reformatted JSON:

Reply:
I just pushed a new commit that, hopefully, addresses this point.